src/GAP/oscar_to_gap.jl

```diff
@@ -0,0 +1,21 @@
+## conversions of Oscar objects to GAP objects
+## (extends the conversions from GAP.jl's `src/julia_to_gap.jl`,
+## where low level Julia objects are treated)
+
+import GAP: julia_to_gap
+
+## WORKAROUND: the following method is also defined in GAP.jl, but apparently
+## we need to redefine it here, otherwise it doesn't work
+GAP.julia_to_gap(obj::Any; recursive::Bool) = GAP.julia_to_gap(obj)
```

Comment:
@ThomasBreuer had to add this to make the tests pass. On the long run, I think we may need to rethink how julia_to_gap and gap_to_julia are organized.

Reply:
@fingolfin I had seen the test failures but the problem was not reproducible in my installation.

Reply:
Since this pull request has been merged, I get warnings during precompilation of Oscar, due to overwritten method definitions.
When I remove the above workaround, these warnings disappear.
The tests run without problems in both situations, with or without the workaround (in Julia 1.4.0 and 1.5.3).